### ISSUE TITLE
Integrate putty

### ISSUE THREAD

**Issue by jirkapok** [Terminals-Origin/Terminals#46]
See also http://terminals.codeplex.com/workitem/35881

**Comment by alxshr** [comment]
Is there an ETA for Putty inegration?

**Comment by Elompenta** [comment]
I like Terminals.. I will use it for SSH Connections too.

Do you have any new informations about putty integration?

**Comment by maishsk** [comment]
Any new updates on this issue?

**Comment by jirkapok** [comment]
Stay tuned, i cant provide any ETA, but this feature is the first you will get.

**Comment by yatishg17** [comment]
teminals is fantastic provided if this issue is fixed with putty integration, as our security will not allow to use diffie-hellman-group1-sha1

**Comment by skewty** [comment]
The putty development branch seems stalled and master has been stagnant even longer.  

Is there a looming announcement that this project is dead and anyone wishing to continue development should fork?

**Comment by jirkapok** [comment]
Yes, there is not progress, but i dont have plans to let it die.

**Comment by joshbouganim** [comment]
@jirkapok  how much work is left in this? if its pretty close, i can give it a go to try and complete it.  i still think terminals is the best application in its class

**Comment by jirkapok** [comment]
Missing:
* Few tests are failing
* database structure needs to be updated - till now there is no update db schema scenario, so it is large implementation part in general
* Something else i dont remember

**Comment by a11eria** [comment]
This project 3 Months away from a year on any notes of progress.  I found this tool a few days ago to help me with connecting to my servers.  I can still try the RDP route on my linux box but ssh appears to be a bust, for now but I am very happy to see that putty is being integrated.  I guess for now I will use some sort of a putty for ssh.
